homeassistant/components/homematicip_cloud/__init__.py

```diff
@@ -213,9 +213,11 @@
 
     def _get_home(hapid: str):
         """Return a HmIP home."""
-        hap = hass.data[DOMAIN][hapid]
+        hap = hass.data[DOMAIN].get(hapid)
         if hap:
             return hap.home
+
+        _LOGGER.info("No matching access point found for access point id %s.", hapid)
```

Comment:
We don't end logging messages with period.

Is this a user message or a message to a developer?

Reply:
it's a user message.